packages/monaco-editor-wrapper/package.json

```diff
@@ -58,10 +58,11 @@
     "npm": "9.8.1"
   },
   "dependencies": {
-    "monaco-languageclient": "~6.4.6"
+    "@codingame/monaco-vscode-configuration-service-override": "~1.82.2",
+    "monaco-languageclient": "6.5.0"
   },
   "peerDependencies": {
-    "monaco-languageclient": "~6.4.6",
+    "monaco-languageclient": "6.5.0s",
```

Comment:
Sorry to disturb, but is this version indented? It has `s` at the end, and there no such version for `monaco-languageclient`:

https://www.npmjs.com/package/monaco-languageclient?activeTab=versions

Reply:
Nope definitely not, that's a mistake that I must've missed when reviewing. Thanks for pointing that out, we can get a patch release on that shortly.

Reply:
@Yokozuna59 thanks for spotting this mistake. Will release a new version in a couple of minutes.

Reply:
@montymxb @kaisalmen No worries. Thanks for your fast response.

Reply:
Done:
https://www.npmjs.com/package/monaco-editor-wrapper/v/3.2.2
https://www.npmjs.com/package/@typefox/monaco-editor-react/v/2.2.2